pv_site_api/convert.py

```diff
@@ -46,20 +46,23 @@ def forecast_rows_to_pydantic_compact(rows: list[Row]) -> ManyForecastCompact:
         start_utc = row.ForecastValueSQL.start_utc
         expected_generation_kw = round(row.ForecastValueSQL.forecast_power_kw, 3)
 
-        if start_utc not in start_utc_idx:
-            start_utc_idx[start_utc] = len(start_utc_idx)
-        idx = start_utc_idx[start_utc]
+        if ~np.isnan(expected_generation_kw):
```

Comment:
What does the tilda do?

Reply:
I feel like `not` is more pythonic, unless there's some edge cases around the `nan` `not` checking that I don't know about...